programs/zstdcli.c

```diff
@@ -1170,6 +1173,17 @@ int main(int const argCount, const char* argv[])
     }   }
 #endif
 
+    if (showDefaultCParams) {
+        if (operation == zom_decompress) {
+            DISPLAY("error : can't use --show-default-cparams in decomrpession mode \n");
+            CLEAN_RETURN(1);
+        }
+        if (!UTIL_isRegularFile(filenames->fileNames[0])) {
```

Comment:
What if there is more than one file?

Reply:
Yeah. good call. I'll check add the check for all files